lib/workers/repository/process/lookup/index.ts

```diff
@@ -538,7 +538,23 @@ export async function lookupUpdates(
             ...config,
             registryUrl: update.registryUrl ?? res.registryUrl,
             lookupName: res.lookupName,
+            currentDigest: config.currentDigest,
           };
+
+          // #20304 only pass it for replacement updates, otherwise we get wrong or invalid digest
+          if (update.updateType !== 'replacement') {
+            delete getDigestConfig.replacementName;
+          }
```

Comment:
@rarkins this is the second fix. we currently always use the `replacementName` for all update types, so the `getDigest` is called with wrong package name for nomal updates and causes digest lookup warnings (especially for docker)

Reply:
Where was it being used after this though?